frontend/src/app/modules/interventions/map/map-filter/map-filter.component.html

```diff
@@ -0,0 +1,10 @@
+<mat-card>
```

Comment:
I didn't have appropriate place to put this comment, so I am putting it here.

Current approach of keeping the card always visible seems to be a little bit problematic from the UX point of view, especially on mobile devices.

The problem:
![ekostraz_map_filter_card_open_intervention_open](https://user-images.githubusercontent.com/15725585/80538660-662ca600-89a6-11ea-92f5-36ad2c1f5b06.png)


Proposed solution - introduce buttons to switch the filters card on/off.

1. Opening - Mini fab button - `<button mini-fab-button>` with filters icon for opening:
![ekostraz_map_filter_faab](https://user-images.githubusercontent.com/15725585/80538034-4fd21a80-89a5-11ea-8174-fac4270d5746.png)

2. After fab click - show card. 
 Note I have also added some more spacing and header styling to have a more proper material design feeling.
Button "Zamknij" I used here is just `<button mat-button>` .

Extra: If you like having fun with new techs and animations, you could try to animate the transition from button to card opening with built in Angular Animations API, I wonder if it is possible

![ekostraz_map_filter_card_open](https://user-images.githubusercontent.com/15725585/80538221-a50e2c00-89a5-11ea-95ef-b479a64689ad.png)

3. After card close we can see the details properly:
![ekostraz_map_filter_fab_intervention_open](https://user-images.githubusercontent.com/15725585/80538333-dab31500-89a5-11ea-964f-0de3b52fa055.png)

Buttons documentation reference:
https://material.angular.io/components/button/overview

What do you think of this proposal? :)

Reply:
Cool, good idea! Added! 👍